test/filter_test.rb

```diff
@@ -262,6 +262,17 @@ class AfterFilterTest < Minitest::Test
     assert_equal 8, count
   end
 
+  it "respects content type set in superclass filter" do
+    base = Class.new(Sinatra::Base)
+    base.before { content_type :json }
+    mock_app(base) do
+      get('/foo'){ {:foo => :bar}.to_json }
```

Comment:
nit: If you don't have any reason to stick to the arrow syntax, could you change this to `foo: :bar`?

Reply:
Will do